### PR TITLE
Update deployment.md

### PR DESCRIPTION
**Pull request by t3db0t**
Added Netlify CD instructions

## React Boilerplate

Thank you for contributing! Please take a moment to review our [**contributing guidelines**](https://github.com/react-boilerplate/react-boilerplate/blob/master/CONTRIBUTING.md)
to make the process easy and effective for everyone involved.

**Please open an issue** before embarking on any significant pull request, especially those that
add a new library or change existing tests, otherwise you risk spending a lot of time working
on something that might not end up being merged into the project.

Before opening a pull request, please ensure:

- [ ] You have followed our [**contributing guidelines**](https://github.com/react-boilerplate/react-boilerplate/blob/master/CONTRIBUTING.md)
- [ ] Double-check your branch is based on `dev` and targets `dev` 
- [ ] Pull request has tests (we are going for 100% coverage!)
- [ ] Code is well-commented, linted and follows project conventions
- [ ] Documentation is updated (if necessary)
- [ ] Internal code generators and templates are updated (if necessary)
- [ ] Description explains the issue/use-case resolved and auto-closes related issues

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

**IMPORTANT**: By submitting a patch, you agree to allow the project
owners to license your work under the terms of the [MIT License](https://github.com/react-boilerplate/react-boilerplate/blob/master/LICENSE.md).

### CODE REVIEW

**Review by julienben — COMMENTED** [review]
This is great! IMO, it would even make sense to have the "Building with Netlify" option simply replace the "Using your own build" one. WDYT?

**Comment by t3db0t** [comment]
Certainly could, I don’t know if there’s any value to doing the build one’s
self X-)

**Comment by julienben** [comment]
Wanna take care of that and then we can merge this then?

**Comment by t3db0t** [comment]
OK—I just committed an update!

File: docs/general/deployment.md [hunk 1]
```diff
@@ -137,20 +137,16 @@ _Step 3:_ Within the root of your react-boilerplate source folder, execute the f
 
 ## Netlify
 
-Netlify is a static site host that deploys from Git repos. To deploy `react-boilerplate` from Netlify:
+Netlify is a static site host that deploys from Git repos. To deploy a `react-boilerplate` app from Netlify, you can leverage their continuous deployment feature. All you have to do is push to your repo and Netlify will [run your production build script](https://docs.netlify.com/configure-builds/get-started/).
 
-_Step 1:_ Clone `react-boilerplate` and remove the default project (`npm run clean`)
+_Step 1:_ Clone react-boilerplate and remove the default project (`npm run clean`)
```
Comment (julienben) [inline docs/general/deployment.md]: Can we improve this step with something like this which points to the quick start instructions?

_Step 1:_ Follow react-boilerplate's [quick start instructions](https://github.com/react-boilerplate/react-boilerplate#quick-start)

File: docs/general/deployment.md [hunk 2]
```diff
@@ -137,20 +137,16 @@ _Step 3:_ Within the root of your react-boilerplate source folder, execute the f
...
 
-_Step 5:_ Comment out the `build` line from your `.gitignore` with a text editor
+_Step 5:_ Go to the **Build & deploy** section of the project settings. For "Build command," enter `npm run build`. For "Publish directory," enter `build`.
 
-_Step 6:_ Build your project (`npm run build`)
-
-_Step 7:_ Commit and push (`git add build`, `git commit -m "first build"`, `git push origin master`
-
-_Step 8:_ Netlify detects the push and automatically redeploys your site.
+_Step 6:_ Commit and push as in Step 7 above. Netlify detects the push, builds the project, and deploys the results.
```
Comment (julienben) [inline docs/general/deployment.md]: Which step 7 does this refer to? For the last step, I would just say: "Update your master branch (either using PRs or by committing directly to it). Netlify will detect both and automatically redeploy your app for you."

**Review by julienben — CHANGES_REQUESTED** [review]

File: docs/general/deployment.md (hunk 1 above)
Comment (t3db0t) [reply docs/general/deployment.md]: Done!

File: docs/general/deployment.md (hunk 2 above)
Comment (t3db0t) [reply docs/general/deployment.md]: Whoops, typo

**Review by julienben — APPROVED** [review]

**Comment by julienben** [comment]
Thanks for your contribution @t3db0t! Will merge this asap.

**Pull request merged**